apocalypse_sim/simulation-pim.py

```diff
@@ -0,0 +1,108 @@
+from model import Apocalypse
```

Comment:
I think this file can be removed in it's entirety, since it doesn't do multithreaded experiments.